tests/Aspire.Hosting.Python.Tests/AddPythonAppTests.cs

```diff
@@ -269,6 +274,18 @@ private static void PreparePythonProject(ITestOutputHelper outputHelper, string
         var requirementsPath = Path.Combine(projectDirectory, "requirements.txt");
         File.WriteAllText(requirementsPath, requirementsContent);
 
+        // This dockerfile doesn't *need* to work but it's a good sanity check.
+        var dockerFilePath = Path.Combine(projectDirectory, "Dockerfile");
+        File.WriteAllText(dockerFilePath,
+        """
+        FROM python:3.9
+        WORKDIR /app
+        COPY requirements.txt .
+        RUN pip install --no-cache-dir -r requirements.txt
+        COPY . .
+        CMD ["python", "main.py"]
+        """);
```

Comment:
```suggestion
        File.WriteAllText(dockerFilePath,
            """
            FROM python:3.9
            WORKDIR /app
            COPY requirements.txt .
            RUN pip install --no-cache-dir -r requirements.txt
            COPY . .
            CMD ["python", "main.py"]
            """);
```

It's more readable if this is indented.